myapp/app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,103 @@
+class TasksController < ApplicationController
+  # Task List
+  def list
+    @tasks = Task.all
+  end
+
+  # Show Task
+  def show
+    @show_task =Task.joins(:user).select('tasks.title, tasks.description, tasks.label, users.name, tasks.status').where(id: params[:id]).first
+  end
+
+  # New Task
+  def new
+    @task = Task.new
+
+    # UserNames
+    @user_names = user_names
+  end
+
+  # New Task → Create Task
+  def create
+    @task = Task.new(task_params)
+    @task.update_attributes(task_params[:task]) unless task_params[:task].blank?
```

Comment:
この行はどのような想定の処理になりますでしょうか？
`update_attributes`メソッドはDBを更新するメソッドであり、26行目の`save`メソッドと細かい処理は異なりますが、役割はほとんど同じです。

また、unlessとblank?の掛け合わせは二重否定なので少々分かりづらいです。（空っぽでなければ、という意味）
判定用メソッド（blank?）に反対の意味のメソッド（present?）が用意されている場合、基本的にはifを利用しましょう。（存在していれば、という意味になり直感的にわかりやすい）
```suggestion
    @task.update_attributes(task_params[:task]) if task_params[:task].present?
```

Reply:
不要だったため削除しました。